src/scriptAPI.js

```diff
@@ -0,0 +1,192 @@
+
```

Comment:
We want to create a modular / object oriented API. (see this link https://developer.mozilla.org/fr/docs/Web/JavaScript/Introduction_à_JavaScript_orienté_objet)